sources/api/views/EventView.ts

```diff
@@ -0,0 +1,16 @@
+import { Event } from "../../model/entity/Event";
+
+export class EventView {
+
+    public static formatEvent(event: Event): any {
+        return {
+            id: event.id,
+            name: event.name,
+            description: event.description,
+            calendar_id: event.calendar.id,
```

Comment:
This means that we always fetch the Calendar along with the Event ?

Reply:
It will be changed in future iterations, with a custom relation between events and calendars